java/lang/Float/Float.hpp

```diff
@@ -494,30 +454,29 @@ namespace Java {
              *
              * @return  a {hash code} value for this object.

```

Comment:
Please fix this comment and check for others comment inside this class

Reply:
Thanks. Fixed